dolphinscheduler-ui/src/views/projects/task/instance/use-table.ts

```diff
@@ -116,6 +116,32 @@ export function useTable() {
         key: 'executorName',
         ...COLUMN_WIDTH_CONFIG['name']
       },
+      {
+        title: t('project.task.operating_environment'),
+        key: 'testFlag',
+        width: 160,
+        render: (row: IRecord) => {
+          if (row.testFlag===0) {
+            return h(
+                NTag,
+                { type: 'success', size: 'small' },
+                {
+                  default: () => t('project.task.on_line')
+                }
+            )
+          } else if(row.testFlag===1){
+            return h(
+                NTag,
+                { type: 'warning', size: 'small' },
+                {
+                  default: () => t('project.task.test')
+                }
+            )
+          }else {
+            return '-'
+          }
+        }
+      },
```

Comment:
It's the same as the testFlag  of the datasource list. It is better to wrap it into a common hook.